File: Day27/guessing_game.py
"""Fixed number to guess."""

GAMENUMBER = 25

""" function to check user number with game number"""

def guess_number():

    """ Ask the user for their guess.."""

    user_number = int(input("Guess a number between 1 to 100 : "))

    #Check the guess number
    if user_number == GAMENUMBER:
        print("Congradulation! You guessed the number.")
    elif user_number > GAMENUMBER:
        print("Too high! Try again.")
        guess_number()
    else:
        print("Too low! Try again.")
        guess_number()

guess_number()

File: Day27/GuessingGame.java
package Day27;
import java.util.Scanner;
public class GuessingGame {

    static int gameNumber = 25;        //Fixed number to guess
    static Scanner sc = new Scanner(System.in);  //Create a global Scanner object to read user input
    static int attempts=0;

    public static void main(String[] args) {
        guessNumber();
        sc.close();
    }
    public static void guessNumber(){

        System.out.println("Guess a number between 1 to 100 :");   //Ask the user for their guess
        int userNumber = sc.nextInt();
        attempts++;

        //Check the guess number
        if( userNumber == gameNumber){
            System.out.println("Congradulation! You guessed the number.");
        }
        else if(userNumber > gameNumber){
            System.out.println("Too high! Try again.");
            guessNumber();
        }else{
            System.out.println("Too low! Try again.");
            guessNumber();
        }
    }
}
